realm-management/libs/import-users.js

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable no-console */
+const ldap = require('ldapjs');
+const fs = require('fs-extra');
+
+const { LDAP_CONFIG, LDAP_CRED, LDAP_USERS_FILE_PATH } = require('../constants');
+
+/**
+ * Import LDAP users to file with GUID attributes
+ * TODO: turn ldap requests into async and await functions
+ */
+const importLdapUsers = async () => {
+  try {
+    /* Read users.json file */
+    const users = JSON.parse(
+      fs.readFileSync(LDAP_USERS_FILE_PATH, { encoding: 'utf-8' }, (err) => {
```

Comment:
doesn't readFileSync return a `Buffer` and not a `String`?

Reply:
a buffer, so need to json parse it